### ISSUE TITLE
Luke Skywalker is trolling (ingame cutscenes broken?)

### ISSUE THREAD

**Issue by hasufell** [JACoders/OpenJK#660]
He just stands in front of his students but doesn't speak a single word. What a troll.

I usually just leave the room then.

**Comment by Razish** [comment]
What level? SP or MP? Latest git revision? OS?

**Comment by hasufell** [comment]
Multiple levels, I'll have to see if it happens again then I'll tell you. SP, latest git revision, OS gentoo. Skipping the cutscenes works fine.

**Comment by hasufell** [comment]
It happens e.g. after the first set of missions when he stands in the hall and explains stuff before the mission to Hoth starts.

**Comment by ensiform** [comment]
Sounds like a compiler issue with some badly written/optimized code maybe?

**Comment by hasufell** [comment]
t1_inter was completely broken, just nothing happened. Eventually after I compiled from source it worked, but I have no idea if that is related.

**Comment by hasufell** [comment]
Same for the academy cutscene before vjun1. Skywalker is still trolling. Other non-academy cutscenes seem to work fine though.

**Comment by who-me** [comment]
Yeah I get this too. First time is easy to repro. It's just after the introductory mission right after we arrive at the academy and he's supposed to speak to the students.

**Comment by kouji-toyota** [comment]
With r_ignoreGLErrors "0" the following is error is thrown:
RE_BeginFrame() - glGetError() failed (0x501)

Here's the first result I got when I googled for this error:
https://forums.geforce.com/default/topic/453170/problems-with-playing-q3a-receiving-an-opengl-error/

Here's my crashlog:
[crashlog-2017-06-18_06-07-07.txt](https://github.com/JACoders/OpenJK/files/1082884/crashlog-2017-06-18_06-07-07.txt)
